CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 * Feat: Send default PII options (#360)
 * Bump: sentry-cocoa to v6.2.1 (#360)
 * Fix: missing event.origin and event.environment tags for non-native events on iOS (#369)
+* Fix: Set `SentryOptions.debug` in `sentry` (#376)
```

Comment:
this PR now mixes up 2 PRs, as we do squash and merge, it's suboptimal, let's be sure in the next PRs to not do this anymore.
We could still use target branches, at the end we should merge them into the main directly, so we guarantee that we have the proper history.
for now, lets update this PRs title and description that it also reflects the `SentryOptions.debug` changes

Reply:
Ah sorry. I didn't think of that. I had set this branch as a target in the other PR so that I can avoid merge conflicts because both PRs touch the same files.

Reply:
no worries, I did this many times :D